.github/workflows/build-test-package.yml

```diff
@@ -2,11 +2,13 @@ name: Build, test, package
 
 on: [push,pull_request]
 
-env:
+inputs:
```

Comment:
This should be kept as `env`. `inputs` is used in GitHub Actions reusable workflows to expose input parameters for calling projects. Here we are not defining a reusable workflow and instead define constant environment variables with `env`.